public/app/features/integrated-alerting/components/AlertRules/AlertRules.constants.ts

```diff
@@ -0,0 +1 @@
+export const ALERT_RULES_TABLE_HASH = 'alert-rules';
```

Comment:
Why is this called `TABLE_HASH` if it doesn't contain an hash?

Reply:
Changed to ID